tox.ini

```diff
@@ -152,15 +154,17 @@ deps =
     behave==1.2.5
 commands =
     # TODO: upgrade behave if they ever take this reasonable PR
+    # starting from setuptools >= 58.0.0, use_2to3 wont be supported
+    # as behave includes use_2to3 in setup.py, it will fail to install
     pip install git+https://github.com/Yelp/behave@1.2.5-issue_533-fork
     pylint -E {toxinidir}/paasta_tools/mesos/ --ignore master.py,task.py
     behave {posargs}
 
 [testenv:mypy]
-basepython = python3.8
+basepython = python3.10
 deps =
     {[testenv]deps}
-    mypy==0.720
+    mypy==0.790
```

Comment:
i dunno why we have a separate tox env for this and don't install mypy in requirements-dev - but that's a later us problem